app/scripts/TrackRenderer.js

```diff
@@ -5,6 +5,7 @@ import { zoom, zoomIdentity } from 'd3-zoom';
 import { select, event, clientPoint } from 'd3-selection';
 import { scaleLinear } from 'd3-scale';
 import slugid from 'slugid';
+import some from 'lodash/some';
```

Comment:
Do we use lodash anywhere else? I.e. do you know if it's already part of the bundle?

If not, I would suggest perhaps just implementing this function so that we're not adding another utility library.

If it is then carry on :-)

Reply:
Oh it was part of package-lock.json. Never mind.